docs/sources/k6-studio/components/test-validator.md

```diff
@@ -61,6 +61,7 @@ If you want to view the script under the **Scripts** section of the k6 Studio ap
 1. Open the k6-studio Scripts folder.
    - For macOS, that can be found in `~/Documents/k6-studio/Scripts`.
    - For Windows, that can be found in `C:\Users\<USERNAME>\Documents\k6-studio`.
+   - For Linux, that can be found in ``.
```

Comment:
@Llandy3d do you know what's the path for Linux here?

Reply:
same as macOS: `~/Documents/k6-studio/Scripts`

I think on Windows there we are missing the last `Scripts` part as well 🤔